templates/inertia.txt

```diff
@@ -16,5 +16,6 @@ export const inertia: InertiaConfig = {
   view: '{{view}}',
   ssr: {
     enabled: {{shouldEnableSsr}},
+    autoreload: process.env.NODE_ENV === 'development',
```

Comment:
I think this should be added only if the use chose to enable SSR:
```
{{#shouldEnableSsr}}
autoreload: process.env.NODE_ENV === 'development',
{{/shouldEnableSsr}}
```
(Not super sure about the syntax here)

Reply:
Just added this 👍